docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,140 @@
+# Scalar DL Backup Creation and Restoration
+
+Scalar DL backup should be created after the Scalar DL server is paused otherwise there is a chance for inconsistency. 
```

Comment:
```suggestion
```

Reply:
Thank you for your suggestion.
Above sentence removed cd32752